### ISSUE TITLE
Craft CMS 4.0

### ISSUE THREAD

**Issue by smcguire1** [Goldinteractive/craft3-publisher#11]
Are there any plans to port/upgrade this wonderful plugin to work on the new Craft CMS 4.0?

Thank you for considering!

![publisher](https://user-images.githubusercontent.com/21061459/167175571-04a90791-2a7e-4c00-88d9-6ebffd446b8b.png)

**Comment by christianruhstaller** [comment]
Yes, we certainly will update it to Craft CMS 4. Unfortunately we weren't able to have a look at it for now. We will update you, as soon as it's ready!

**Comment by masonry47** [comment]
+1. Just seeing if there is any sort of a timeline on this.  Thanks so much!

**Comment by christianruhstaller** [comment]
For now we planned that we release it in the next few weeks. But as a heads up it will most likely end up as a paid plugin.

**Comment by mark-chief** [comment]
+1 yes please, we are starting to move to Craft 4 now and this is one of the final plugins for us. :)

**Comment by christianruhstaller** [comment]
We try to release this plugin this or next week. Stay tuned.

**Comment by weotch** [comment]
Any updates? 🤞

**Comment by eduardoalonsoalbella** [comment]
Any updates?

**Comment by smcguire1** [comment]
Goldinteractive has created a new repository for the new version: Publisher X

https://github.com/Goldinteractive/craft-publisher

Thank you!

**Comment by janreges** [comment]
+1
